resolwe_bio/tests/processes/test_genome_browser.py

```diff
@@ -0,0 +1,26 @@
+# pylint: disable=missing-docstring
+from resolwe_bio.utils.test import BioProcessTestCase
+
+
+class GenomeBrowserProcessorTestCase(BioProcessTestCase):
+
+    def test_igv_bam(self):
+        bam = self.prepare_bam()
+
+        inputs = {'src': 'reads.bam'}
+        bam1 = self.run_process('upload-bam', inputs)
+
+        inputs = {
+            'genomeid': 'hg19',
+            'bam': [bam.pk, bam1.pk],
+            'locus': 'chr7:79439229-79481604'
+        }
+
+        igv_session = self.run_process('igv', inputs)
+
+        # remove changing lines from the output
+        def filter_created(line):
+            if b'<Resource' in line:
```

Comment:
This is too broad, since it will also match line ` <Resources>` in `resolwe_bio/tests/files/igv_session_bam.xml`. Try to be as specific as possible when you are filtering out lines to compare.

I would use something like: `if b'<Resource path=' in line:`.
Also, change filter's name to match what it is actually doing. I would suggest something like `filter_resource`.